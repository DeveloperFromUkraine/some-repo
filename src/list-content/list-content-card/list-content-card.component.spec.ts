import { ComponentFixture, TestBed } from "@angular/core/testing";
import { ListContentCardComponent } from "index";
import { ComponentTest } from "../../../test/test-bed/component";
import { MatCardModule } from "@angular/material";

describe('ListContentCardComponent', () => {
    let fixture: ComponentFixture<ListContentCardComponent>;

    beforeEach(async () => {
        ComponentTest.createTestBed([MatCardModule], [ListContentCardComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(ListContentCardComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});