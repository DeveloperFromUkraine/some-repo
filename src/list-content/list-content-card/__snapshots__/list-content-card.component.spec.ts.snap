// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Content Card should match snapshot 1`] = `
<ign-list-content-card>
  <mat-card
    class="container mat-card"
  >
    
    
    
  
  </mat-card>
  
</ign-list-content-card>
`;
